homeassistant/components/media_player/reproduce_state.py

```diff
@@ -101,12 +107,6 @@ async def call_service(service: str, keys: Iterable) -> None:
     ):
         await call_service(SERVICE_SELECT_SOURCE, [ATTR_INPUT_SOURCE])
```

Comment:
We should do the same with source probably ?

Reply:
I missed that, I assume source should be the first one, than sound mode (for webos tv it doesn't matter, but I can guess some devices may have different sound outputs for different sources), Is it OK to put it first?

Reply:
Sure.